tests/test_config.py

```diff
@@ -1,17 +1,15 @@
 from pathlib import Path
 
-from lobbyboy.config import load_config
-from lobbyboy.main import load_providers
+from lobbyboy.config import LBConfig
 
 PARENT_DIR = Path(__file__).parent
 CONFIG_FILE = PARENT_DIR.parent / "lobbyboy" / "conf" / "lobbyboy_config.toml"
 
 
 def test_load_config():
-    load_config(CONFIG_FILE)
+    LBConfig.load(CONFIG_FILE)
 
 
 def test_load_providers_from_config():
-    config = load_config(CONFIG_FILE)
-    providers = load_providers(config.provider, PARENT_DIR.parent / "dev_datadir")
-    assert len(providers) > 0
+    config = LBConfig.load(CONFIG_FILE)
+    assert len(config.provider_cls) > 0
```

Comment:
This test slightly changed behavior, previous it will actually initialize the provider class in `load_providers`.

Reply:
Yeah, current test state is not perfect, and this PR seems to reduce the test cov. I'll improve the test next time.